mac

```diff
@@ -137,36 +146,18 @@ brew "pipenv"
 brew "tfenv"
 
 # Databases
-brew "postgres", restart_service: :changed
+brew "postgresql@14", restart_service: :changed
 brew "redis", restart_service: :changed
 
 # Development tools
 brew "awscli"
-brew "heroku/brew/heroku"
 brew "getsentry/tools/sentry-cli"
 
-# Applications
-cask "google-chrome"
-cask "firefox"
-cask "slack"
-cask "iterm2"
-cask "sublime-text"
-cask "sublime-merge"
-cask "visual-studio-code"
-cask "insomnia"
-cask "flycut"
-cask "bitwarden"
-cask "caffeine"
-cask "docker"
-cask "zoom"
-cask "tableplus"
-EOF
```

Comment:
Open to debate here. I can see arguments in any direction. My impression is that none of these applications would tell you that the correct way to install them is via Homebrew. (Especially not Zoom). But I'm fine with them being here especially if you @rossettistone feel strongly.